boxoffice/views/admin.py

```diff
@@ -24,11 +25,26 @@ def index():
     return dict(user=g.user)
 
 
+def jsonify_ic(ic):
+    if ic:
+        return {
+            'id': ic.id,
+            'name': ic.name,
+            'title': ic.title,
+            'url': url_for('admin_item_collection', ic_id=ic.id),
+            'description_text': ic.description_text,
+            'description_html': ic.description_html
+        }
+
+
 def jsonify_org(data):
     item_collections_list = ItemCollection.query.filter(ItemCollection.organization == data['org']).order_by('created_at desc').all()
+    form = ItemCollectionForm()
+    html_form = render_form(form=form, title=u"New Item Collection", submit=u"Save", ajax=False, with_chrome=False)
     return jsonify(id=data['org'].id,
         org_title=data['org'].title,
-        item_collections=[{'id': ic.id, 'name': ic.name, 'title': ic.title, 'url': '/ic/' + unicode(ic.id), 'description_text': ic.description_text, 'description_html': ic.description_html} for ic in item_collections_list])
+        item_collections=[jsonify_ic(ic) for ic in item_collections_list],
```

Comment:
`item_collections=[ic.access_for(…) for ic in item_collections_list]` @shreyas-satish

Reply:
Fixed